source/release-notes/1.6.txt

```diff
@@ -0,0 +1,34 @@
+.. _c2c-release-notes-1.6:
+
+===================================
+Release Notes for mongosync 1.6
+===================================
+
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. _1.6.0-c2c-release-notes:
+
+1.6.0 Release
+-------------
+
+** TBD **
```

Comment:
**Suggestion**

Update this too `Upcoming` to match the style of server release notes

```suggestion
** Upcoming **
```